gcs/src/main/java/com/google/cloud/hadoop/fs/gcs/GoogleHadoopFileSystemBase.java

```diff
@@ -423,11 +423,12 @@ public ParentTimestampUpdateIncludePredicate(
 
     /**
      * Determine if updating directory timestamps should be ignored.
-     * @return True if the directory timestamp should not be updated. False to indicate it should
-     * be updated.
+     *
+     * @return True if the directory timestamp should not be updated. False to indicate it should be
+     *     updated.
      */
     @Override
-    public boolean shouldUpdateTimestamp(URI uri) {
+    public boolean test(URI uri) {
```

Comment:
Why do we rename it? I feel "test" is a bad name, unclear what it actually means.

Reply:
This is a `test` method in `java.util.function.Predicate` - I just removed custom predicate class and switched to provided by JDK